COMPACTION_POLICY - added warning (#1195)
redis/docs: content/develop/data-types/timeseries/configuration.md

  Assure that there is a bucket that starts at exactly _alignTimestamp_ after the epoch and align all other buckets accordingly. Default value: 0 (aligned with the epoch). Example: if _bucketDuration_ is 24 hours, setting _alignTimestamp_ to `6h` (6 hours after the Epoch) will ensure that each bucket’s timeframe is [06:00 .. 06:00).

{{% warning %}}

> On a clustered environment, if `COMPACTION_POLICY` is set, you must use [hash tags]({{< relref "/operate/oss_and_stack/reference/cluster-spec" >}}#hash-tags) for all time series key names. 
> This will force each compaction to be created in the same hash slot as its source key.
> Failure to do so can result in data not being compacted with no error messages.

Reviewer: Active voice copy-editing.

```suggestion
In a clustered environment, if you set `COMPACTION_POLICY`, you must use [hash tags]({{< relref "/operate/oss_and_stack/reference/cluster-spec" >}}#hash-tags) for all time series key names. This ensures that the system creates each compaction in the same hash slot as its source key. If you don't, the system may fail to compact the data without displaying any error messages.
```

  Assure that there is a bucket that starts at exactly _alignTimestamp_ after the epoch and align all other buckets accordingly. Default value: 0 (aligned with the epoch). Example: if _bucketDuration_ is 24 hours, setting _alignTimestamp_ to `6h` (6 hours after the Epoch) will ensure that each bucket’s timeframe is [06:00 .. 06:00).

{{% warning %}}

> In a clustered environment, if you set `COMPACTION_POLICY`, you must use [hash tags]({{< relref "/operate/oss_and_stack/reference/cluster-spec" >}}#hash-tags) for all time series key names. This ensures that Redis will create each compaction in the same hash slot as its source key. If you don't, the system may fail to compact the data without displaying any error messages.

Author: there is one additional "the system" ;-)

Reviewer: It's already been merged; I'll address in a later PR.

  Assure that there is a bucket that starts at exactly _alignTimestamp_ after the epoch and align all other buckets accordingly. Default value: 0 (aligned with the epoch). Example: if _bucketDuration_ is 24 hours, setting _alignTimestamp_ to `6h` (6 hours after the Epoch) will ensure that each bucket’s timeframe is [06:00 .. 06:00).

{{% warning %}}

> In a clustered environment, if you set `COMPACTION_POLICY`, you must use [hash tags]({{< relref "/operate/oss_and_stack/reference/cluster-spec" >}}#hash-tags) for all time series key names. This ensures that the system creates each compaction in the same hash slot as its source key. If you don't, the system may fail to compact the data without displaying any error messages.

Author: Thanks. Maybe change "the system" to "Redis" (or find another way to put it)?

Reviewer: Done. Thanks.
